### ISSUE TITLE
'Art Vandelay' import/export error

### ISSUE THREAD

**Issue by jehillert** [styled-components/styled-components#2484]
Apologies for taking up your time. I spent two hours writing this question up on stack overflow, but when I clicked submit it said I had insufficient rep to post to 'styled-components'.  I had to post it to to general JS/React tags.  Could you take a look and point me in the right direction?  

https://stackoverflow.com/questions/55552315/error-when-styling-react-components-imported-from-index-file-but-not-when-impor

The URL is not cut off, it just looks that way.
Many thanks.
John

**Comment by probablyup** [comment]
Looks like you got a response on SO, closing as this is not a general help forum.

**Comment by jehillert** [comment]
Uh, yes.  Somebody added me to the styled-components tag and then suggested
an answer that was really helpful.  Sorry for bugging you guys.  If
 something like this comes up again, I will see if I can get a SO mod to
help me out instead.

Btw, thank you for doing what you do.  I’ve been spending most of my time
over the last year trying to teach myself how to develop web applications.
One of the pain points has been figuring out which approach to styling to
focus on. There’s a lot to choose from, and I found many ways of
implementing css to be tedious.  Then I ran across styled-components and
haven’t looked back.  It’s fun to implement, logical, straightforward,
powerful, and has made it a lot easier to learn css.  It has also made it a
lot easier to understand how some of the bigger UI Libraries like Material
UI are implemented.   So again, thanks much.

John
